DANS-KNAW/easy-update-metadata-dataset
# Plain ascii

fixes EASY-

#### When applied it will

* accept plain ascii files
* have tests to proof it accepts both plain ascii files as UTF-8
* 

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

* Create files that changes some EMD field like a title, description or name of a contributor, see the README
  * Create the file with commonly used editors such as `vi` or an export of a spread sheet from open office
  * Experiment with the default save method and save as UTF-8
  * Try characters within and beyond [ISO-8859](https://en.wikipedia.org/wiki/ISO/IEC_8859)-1
* Run these files in test mode and check for question marks replacing non-ascii characters in the logging
 

#### related pull requests on github
repo                       | PR | Note
-------------------------- | ----------------- | ---
 .. | #5 | the new check for UTF-8 files disabled plain ascii files
easy-split-multi-deposit                      | n.a. | should apply the AutoDectectReader too, this project serves as a pilot
